src/avalanche/test/proof_tests.cpp

```diff
@@ -635,7 +635,8 @@ BOOST_AUTO_TEST_CASE(deserialization) {
         BOOST_CHECK_EQUAL(p.getScore(), c.score);
 
         ProofValidationState state;
-        BOOST_CHECK_EQUAL(p.verify(PROOF_DUST_THRESHOLD, state),
+        // TODO Redo test cases using new PROOF_DUST_THRESHOLD
```

Comment:
```suggestion
        // TODO: Redo test cases using new PROOF_DUST_THRESHOLD
```

can you add some more instructions for a future dev in case this gets lost to time?